WebAutopark.DataBaseAccess/Repository/Base/IRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace WebAutopark.DataBaseAccess.Repository.Base
+{
+    interface IRepository<T> 
+        where T : class
+    {
+        public IEnumerable<T> GetAllItems();
```

Comment:
```suggestion
        IEnumerable<T> GetAllItems();
```
It`s redundant at IRepository, because all interface methods public by default

Reply:
Fixed